redirector/redirector.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+import common
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+
+def lambda_handler(event, context):
+    filename = event["pathParameters"]["filename"]
+
+    LOGGER.info("Received request for %s", filename)
+
+    ddb_table = common.get_ddb_table()
+    download_url = common.get_download_url(ddb_table, filename)
+
+    if download_url:
```

Comment:
Is it pythonic to use None as equivalent to False?

Reply:
I guess I don't like that, if `download_url==None`, `not download_url` returns `True`, but any comparison of `download_url` to `False` returns `False`. Maybe this is a totally pythonic way to deal with NoneType's and I've just never thought about it before.

Reply:
I suppose since NoneType's evaluate to False it's by definition pythonic.

Reply:
Changing to `is None` per discussion on Slack